.github/workflows/import-l10n.yml

```diff
@@ -0,0 +1,78 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+name: Import l10n translations
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  schedule:
+    - cron: "10 0 * * *"
```

Comment:
I realized afterwards that this might be running 10 minutes after the current TaskCluster workflow.
https://github.com/mozilla-l10n/android-l10n-tooling/blob/master/.cron.yml#L19

Depending on how we approach this (keep them running in parallel for a few days), we might want to run this one a couple of hours later.

Reply:
Updated this to “0 3 * * *”